src/models/Facebook.js

```diff
@@ -23,10 +23,8 @@ define(['ShareToolsModel'], function (ShareToolsModel) {
             'redirect_uri': 'http://www.bbc.co.uk/news/special/shared/vj_sharetools/fb_red_uri.html?st_cb=facebook#state=feed',
             'display':      'popup',
             'locale':       'en_GB',
-            'link':         this.getShareUrl(),
-            'name':         this.getMessage().title,
-            'description':  this.getMessage().description,
-            'picture':      this.getMessage().image
```

Comment:
Hi - do we not need to pass picture and description anymore? Does picture now come from the opengraph data in the parent page?

Reply:
Picture and description are deprecated. I think the open graph tags do the work now.

Reply:
Great - please merge, and bump the version number, `git tag` and also `npm publish`: https://www.npmjs.com/package/@bbc/news-vj-sharetools

Thanks!